x-pack/platform/plugins/private/upgrade_assistant/public/application/components/es_deprecations/deprecation_types/indices/flyout/steps/details/unfreeze_details_step.tsx

```diff
@@ -214,7 +226,8 @@ export const UnfreezeDetailsFlyoutStep: React.FunctionComponent<{
           </EuiFlexItem>
           <EuiFlexItem grow={false}>
             <EuiFlexGroup gutterSize="s">
-              {!hasFetchFailed && !isCompleted && hasRequiredPrivileges && (
+              {/* We cannot unfreeze backing indices (that would break the related data_stream) */}
```

Comment:
```suggestion
              {/* We cannot reindex backing indices in the same way as regular indices (that would break the related data_stream) */}
```